profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc

```diff
@@ -592,7 +592,7 @@ xsi:noNamespaceSchemaLocation=\"http://intragate.ec.europa.eu/DGT/poetry_service
     try {
       $client = new SoapClient($poetry['address'], array('cache_wsdl' => WSDL_CACHE_NONE));
 
-      $response = $client->$poetry['method']($poetry_user, $poetry_password, $msg);
+      $response = $client->{$poetry['method']}($poetry_user, $poetry_password, $msg);
```

Comment:
`return $client->{$poetry['method']}($poetry_user, $poetry_password, $msg);`

Reply:
Thank you. 
Fixed.

Reply:
thanks mate, perfect :-)